src/Navbar.test.js

```diff
@@ -43,10 +43,13 @@ describe("Navbar", () => {
     expect(wrapper.find(".login").length).toBe(0);
     expect(wrapper.find(".profilePage").get(0).props.children).toBe("Profile");
     expect(wrapper.find(".logout").get(0).props.children).toBe("Logout");
```

Comment:
If you create a util file for the `findElement` function you created, you can import it and reuse it here as well

Reply:
Took a note of this, will bring this up as a ticket and address these changes during our technical debt stage